src/config_cubes.c

```diff
@@ -254,12 +254,13 @@ TbBool load_cubes_config_file(const char *textname, const char *fname, unsigned
             WARNMSG("The %s file \"%s\" doesn't exist or is too small.",textname,fname);
         return false;
     }
-    if (len > MAX_CONFIG_FILE_SIZE)
```

Comment:
Would it be prudent to set a new max here, that's sensible for configs how we do want them? If so, perhaps INT_MAX over SHRT_MAX it is now.
If not, we can remove the entire thing instead if commenting it out.